controllers/drplacementcontrol_controller.go

```diff
@@ -142,7 +161,10 @@ var (
 
 func init() {
 	// register custom metrics with the global Prometheus registry
-	metrics.Registry.MustRegister(failbackTime.gauge, failoverTime.gauge, relocateTime.gauge, deployTime.gauge)
+	metrics.Registry.MustRegister(failoverTime.gauge, failoverTime.histogram)
+	metrics.Registry.MustRegister(failbackTime.gauge, failbackTime.histogram)
```

Comment:
You need to rebase.  There is no failback.  We only have now Failover and Relocate

Reply:
Rebase is done, and the Failback metric has been removed